CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Cancel command for the ASB that allows cancelling a specific swap by id.
+  Using the cancel command requires the cancel timelock to be expired, but `--force` can be used to circumvent this check.
```

Comment:
I wanted to keep the commands somewhat similar, but that is probably not a good idea :D
I can think about it - the thing is that for some commands `--force` disables not only the timelocks but also allows executing the command from a state that is not directly associated with the command.

I will re-visit this on Monday so I can give it some more thought :)

Reply:
I am not sure about this one yet, I would keep `--force` for now and re-visit it once we get feedback from users. In the end these commands are power²-user commands anyway

IMO adding different flags for each command is not very user-friendly neither. 
The idea is that `--force` circumvents all checks that CAN be circumvented because they are not a hard dependency. And that is the same for all commands. 

Hard dependency are dependencies on e.g. adapter signature publication,  e.g. for Alice's refund, Bob must have published refund as well, that cannot be circumvened)

Reply:
You can't also really circumvent the timelock right? :)

What this shows is that we should likely not even make this check ourselves but just try and broadcast it and handle the specific errors better.

Reply:
And then this flag just goes away for this command, even simpler!